src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -1132,6 +1132,15 @@ private function addNewInstance(Definition $definition, string $return = '', str
         if (null !== $definition->getFactory()) {
             $callable = $definition->getFactory();
 
+            if (['Closure', 'fromCallable'] === $callable && [0] === array_keys($definition->getArguments())) {
+                $callable = $definition->getArgument(0);
+                $arguments = ['...'];
```

Comment:
will this dump a first-class callable or a string containing `...` ?

Reply:
it will dump a first-class callable, because this is set after values are dumped to string (see fixture in tests as a proof ;) )